Sources/PointFree/Account/Invite.swift

```diff
@@ -163,12 +165,41 @@ let sendInviteMiddleware =
 
       let (email, inviter) = lower(conn.data)
 
-      return Current.database.insertTeamInvite(email, inviter.id)
+      let seatsTaken = zip2(
```

Comment:
Sending invites was missing some validation that was only being done on the frontend. This endpoint will now check to see how many seats are available before sending another invite.